knit/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 from .utils import *
 from .core import *
-from .env import CondaCreator
+from .env import CondaCreator, zip_path
```

Comment:
Why add this to the top-level namespace?

Reply:
It seems like a generally useful function, e.g., for what you are doing with folder zipping. The lines above tend to import everything.